client/lib/login/index.js

```diff
@@ -149,6 +137,8 @@ export function getSignupUrl( currentQuery, currentRoute, oauth2Client, locale,
 			oauth2_client_id: oauth2Client.id,
 			oauth2_redirect: redirectTo,
 		} );
+
+		const wccomFrom = get( currentQuery, 'wccom-from' );
```

Comment:
Here I'm moving the variable assignment closer to the (single) usage. Similarly for `signupFlow` a few lines below.